precompiles/ArbOwner.go

```diff
@@ -57,6 +61,46 @@ func (con ArbOwner) GetAllChainOwners(c ctx, evm mech) ([]common.Address, error)
 	return c.State.ChainOwners().AllMembers(65536)
 }
 
+// AddChainOwner adds account as a native token owner
+func (con ArbOwner) AddNativeTokenOwner(c ctx, evm mech, newOwner addr) error {
+	if c.State.ArbOSVersion() < params.ArbosVersion_41 {
```

Comment:
(note: Acces to ArbosState has code that deals with burning Gas as appropriate, so it doesn't need to be done explicitly)

Reply:
Noted.